### PR TITLE
updating  links on web-ext docs

### LINKED ISSUE #355: Update links on web-ext command reference page
Filing this issue to make sure it stays on radar; needs specific details.

### PR DESCRIPTION
**Pull request by caitmuenster**
Fixes #355 and #354.

This PR should not have a ton of old commits in it.

### CODE REVIEW

File: documentation/develop/getting-started-with-web-ext.md [hunk 1]
```diff
@@ -185,11 +185,11 @@ You can connect to this Android device on TCP port 51499
 
 This is a remote debugger port that you can [connect to with Firefox's developer tools](https://developer.mozilla.org/en-US/docs/Tools/Remote_Debugging/Firefox_for_Android#Connecting). In this case, you'd connect to host `localhost` on port `51499`.
```
Comment (championshuttler) [inline documentation/develop/getting-started-with-web-ext.md]: Should not we remove the locale from the rest of the urls in this file or need to file a bug to remove all locale at once from the project @caitmuenster

Comment (caitmuenster) [reply documentation/develop/getting-started-with-web-ext.md]: 👍 Yes, good catch -- addressing it in next commit

File: documentation/develop/getting-started-with-web-ext.md [hunk 2]
```diff
@@ -183,13 +183,13 @@ When using `web-ext run` to test an extension on Firefox for Android, you'll not
...
+When you execute [web-ext run](/documentation/develop/web-ext-command-reference/#web-ext-run), the extension gets installed temporarily until you close Firefox. This does not violate any signing restrictions. If instead you create a zip file with [web-ext build](/documentation/develop/web-ext-command-reference/#web-ext_build) and try to install it into Firefox, you will see an error telling you that the add-on is not signed. You will need to use an [unbranded build](https://wiki.mozilla.org/Addons/Extension_Signing#Unbranded_Builds) or use a [development build](https://www.mozilla.org/firefox/developer/) to install unsigned extensions.
```
Comment (championshuttler) [inline documentation/develop/getting-started-with-web-ext.md]: Can you please also change the `/documentation/develop/web-ext-command-reference/#web-ext_build` to `/documentation/develop/web-ext-command-reference/#web-ext-build` as per

Comment (caitmuenster) [reply documentation/develop/getting-started-with-web-ext.md]: 👍

**Pull request merged**